src/utils/jwt.encryption.ts

```diff
@@ -0,0 +1,50 @@
+import { randomBytes } from 'crypto'
+import { JWE, JWK } from 'node-jose'
+
+import { AuthConfig } from 'app/middleware/companyAuth.middleware'
```

Comment:
This is a circular dependency. CompanyAuthMiddleware calls this file, and this file calls the CompanyAuthMiddleware file